extensions/blocks/instagram-gallery/edit.js

```diff
@@ -58,22 +60,37 @@ const InstagramGalleryEdit = props => {
 		apiFetch( {
 			path: addQueryArgs( '/wpcom/v2/instagram/gallery', {
 				access_token: accessToken,
-				count,
+				count: 30,
 			} ),
-		} ).then( response => {
+		} ).then( ( { external_name: externalName, images: imageList } ) => {
 			setIsLoadingGallery( false );
 
-			if ( isEmpty( response.images ) ) {
+			if ( isEmpty( imageList ) ) {
 				noticeOperations.createErrorNotice(
 					__( 'No images were found in your Instagram account.', 'jetpack' )
 				);
 				return;
 			}
 
-			setAttributes( { instagramUser: response.external_name } );
-			setImages( response.images );
+			setAttributes( { instagramUser: externalName } );
+			setImages( imageList );
 		} );
-	}, [ accessToken, count, noticeOperations, setAttributes ] );
+	}, [ accessToken, noticeOperations, setAttributes ] );
```

Comment:
I'm not sure we need `noticeOperations` and `setAttributes` here

Reply:
We might not, but ESLint would complain. 😄 
See: https://github.com/facebook/react/issues/14920